src/multiworld.cpp

```diff
@@ -0,0 +1,77 @@
+
+#include <climits>
+#include <cstdint>
+#include <iosfwd>
+#include <new>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include <multiworld.h>
+#include <map.h>
+#include <avatar.h>
+#include <game.h>
+#include <weather.h>
+#include <mapbuffer.h>
+#include <overmapbuffer.h>
+
+multiworld::multiworld() = default;
+multiworld::~multiworld() = default;
+multiworld MULTIWORLD;
+
+void multiworld::set_world_prefix( std::string prefix )
+{
+    world_prefix = std::move( prefix );
```

Comment:
Minor nit, but don't use std::move() unless you have a specific reason to.

Reply:
that one was clang-tidy iirc 
sure though

Reply:
the heck, clang-tidy shouldn't be telling people to std::move stuff pointlessly :/